packit_service/worker/parser.py

```diff
@@ -1292,3 +1295,25 @@ def parse_pagure_pull_request_comment_event(
             comment=comment,
             comment_id=comment_id,
         )
+
+    @staticmethod
+    def parse_new_hotness_update_event(event) -> Optional[NewHotnessUpdateEvent]:
+        if "hotness.update.bug.file" not in event.get("topic", ""):
+            return None
+
+        package_name = event.get("package")
+        dg_base_url = getenv("DISTGIT_URL", PROD_DISTGIT_URL)
+
+        distgit_project_url = f"{dg_base_url}rpms/{package_name}"
```

Comment:
I don't want to break more things than needed, but wouldn't be better to have the `/` here, instead of the constant/env-var?

Reply:
it probably would, but this is already used across repositories, [see](https://github.com/search?p=1&q=org%3Apackit+DISTGIT_URL&type=Code), so changing the constant would not be enough

Reply:
I mainly meant the constant but you are right that we should be consistent between the config and env.var...